sos/report/plugins/aap_hub.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Mike Silmser <msilmser@redhat.com>
+# Copyright (c) 2024 Lucas Benedito <lbenedit@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPAutomationHub(Plugin, RedHatPlugin):
+    short_desc = 'AAP Automation Hub plugin'
+    plugin_name = 'aap_hub'
+    profiles = ('sysmgmt', 'ansible')
+    packages = ('automation-hub')
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/ansible-automation-platform/",
+            "/var/log/ansible-automation-platform/hub/worker.log*",
+            "/var/log/ansible-automation-platform/hub/pulpcore-api.log*",
+            "/var/log/ansible-automation-platform/hub/pulpcore-content.log*",
+            "/var/log/nginx/automationhub.access.log*",
+            "/var/log/nginx/automationhub.error.log*",
```

Comment:
We should add to [nginx plugin](https://github.com/sosreport/sos/blob/main/sos/report/plugins/nginx.py#L36-L37) a prohibition to collect these files, like:

            if self.get_option("log") or self.get_option("all_logs"):
                self.add_copy_spec("/var/log/nginx/*")
                self.add_forbidden_path(["/var/log/nginx/automationhub.access.log*", "/var/log/nginx/automationhub.error.log*"])

(cf. https://github.com/sosreport/sos/blob/main/sos/report/plugins/apache.py#L63-L65 for reasoning / example)

Reply:
@pmoravec shouldn't the default behavior already collect all the logs? We only added to this plugin because we never receive the necessary logs when a customer collects it

Reply:
Did you run `sos report` with option `--all-logs`? (Only) then `nginx plugin will collect the `automationhub*` files.

Reply:
submitted #3568 for excluding them from nginx logs